Core/App/AppInstaller.php

```diff
@@ -356,23 +354,28 @@ private function testMysql($dbData)
     private function testPostgreSql($dbData)
     {
         $connectionStr = 'host=' . $dbData['host'] . ' port=' . $dbData['port'];
-        $connection = \pg_connect($connectionStr . ' user=' . $dbData['user'] . ' password=' . $dbData['pass']);
-        if ($connection) {
-            // Check that the DB exists, if it doesn't, we create a new one
-            $connection2 = \pg_connect($connectionStr . ' dbname=' . $dbData['name'] . ' user=' . $dbData['user'] . ' password=' . $dbData['pass']);
-            if ($connection2) {
+        $connection = @\pg_connect($connectionStr . ' dbname=postgres user=' . $dbData['user'] . ' password=' . $dbData['pass']);
+        if (is_resource($connection)) {
```

Comment:
On Postgres I haven't got enough experience, but talking about this with José Cuello, seems that this change looks better (maybe needs more improvements, he explain me that this was the correct way to do that).

Don't require privileged user to create a new database (I don't found any issue about this, but José explain me that he was), now if user can connect to their own postgres db, we can check the same without a second db connection, and create the database in case was needed.

If you want to try this one, create a new user in your postgres without privileges, and try it before apply this changes and after. The first try must fail, and second must success.